abc/src/base/abci/abcTiming.c

```diff
@@ -45,7 +45,7 @@ struct Abc_ManTime_t_
 
 #define TOLERANCE 0.001
 
-static inline int          Abc_FloatEqual( float x, float y )     { return fabs(x-y) < TOLERANCE; }
+static inline int          Abc_FloatEqual( float x, float y )     { return fabsf(x-y) < TOLERANCE; }
```

Comment:
The abc/ directory is probably also something that is just imported and we shouldn't touch too much.